styles/go.dev/catppuccin.user.css

```
@@ -0,0 +1,938 @@
+/* ==UserStyle==
+@name go.dev Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/go.dev
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/go.dev
+@version 1.0.0
```

```suggestion
@version 0.0.1
```

As the template suggests, let's keep it at an `0.0.1` version number.